sql/hive/src/test/scala/org/apache/spark/sql/hive/client/FiltersSuite.scala

```diff
@@ -63,6 +64,24 @@ class FiltersSuite extends SparkFunSuite with Logging with PlanTest {
     (Literal(1) === a("intcol", IntegerType)) :: (Literal("a") === a("strcol", IntegerType)) :: Nil,
     "1 = intcol and \"a\" = strcol")
 
+  filterTest("date filter",
```

Comment:
do we run these test with different hive versions?

Reply:
Different hive versions tested by `HivePartitionFilteringSuite`:
https://github.com/apache/spark/blob/ba2f5531601ff169ff0f9fab9bb60627f65082bf/sql/hive/src/test/scala/org/apache/spark/sql/hive/client/HivePartitionFilteringSuite.scala#L290-L343